packages/flutter/test/semantics/semantics_test.dart

```diff
@@ -204,9 +206,8 @@ void main() {
 
     final SemanticsConfiguration config = new SemanticsConfiguration()
       ..isMergingSemanticsOfDescendants = true
-      ..addAction(SemanticsAction.scrollUp, () { })
-      ..addAction(SemanticsAction.longPress, () { })
-      ..addAction(SemanticsAction.showOnScreen, () { })
```

Comment:
What happened to `showOnScreen`?

Reply:
Done

Reply:
I brought `showOnScreen` back. Initially, I though I would omit it since it is more of an internal action. However, somebody who wants to go fully custom might want to override the action (just like we do on Android).